src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,76 @@
+package duke;
+
+import duke.command.Command;
+import duke.ui.Ui;
+
+/**
+ * Main class for the duke application.
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+    private boolean isExit;
+
+    public Duke() {
+        this("data/duke.txt");
```

Comment:
If the file path is constant information you can define it as a constant variable